UnityProjects/MRTKDevTemplate/Assets/Scenes/ClippingExamples.unity

```diff
@@ -2528,7 +2528,199 @@ MeshRenderer:
   m_SortingLayer: 0
   m_SortingOrder: 0
   m_AdditionalVertexStreams: {fileID: 0}
---- !u!21 &1269446260
+--- !u!1001 &1288658493
```

Comment:
@ms-RistoRK can you check this scene.   This looks like a lot more changes than just enabling the Speech Interactor...

Reply:
Double checked the scene, those extra changes in file were introduced by Unity Editor.  Just in case I repeated the process of copying .unity file from feature/XRI3 branch latest commit -> double-checked the new controllerless rig is used -> enabled MRTKSpeech rig component.
Resolving.